test/sql/test_compaction_cron/R/test_compaction_cron

```diff
@@ -1,4 +1,72 @@
--- name: test_create_alter_compaction_control
+-- name: test_disable_compaction @sequential
+create database test_disable_compaction;
+-- result:
+-- !result
+use test_disable_compaction;
+-- result:
+-- !result
+create table t(k int, v int) properties('base_compaction_forbidden_time_ranges'='* * * * *');
+-- result:
+-- !result
+update information_schema.be_configs set value='-1' where name='disable_compaction_tablet_size_threshold';
```

Comment:
need to reset it to default value, avoid affecting subsequent cases even the sql tester is running in sequential.

Reply:
Restoring it to the original value does not guarantee that the parallel execution case will be executed correctly, so the only way is set this value before the execution and execute it serially.

Reply:
yes. it has to be set to sequential and also restore the parameter after the case is done. Be respect to the test env, leave a clean env after the case is done.